libevmasm/ConstantOptimiser.h

```diff
@@ -130,6 +133,8 @@ class ComputeMethod: public ConstantOptimisationMethod
 		ConstantOptimisationMethod(_params, _value)
 	{
 		m_routine = findRepresentation(m_value);
+		if (!checkRepresentation(m_value))
+			BOOST_THROW_EXCEPTION(AssemblyException());
```

Comment:
Is this exception available on the higher levels or it is hidden within `InternalCompilerException`?